plugincontainer/examples/container/Dockerfile

```diff
@@ -7,14 +7,15 @@ COPY go-plugin-counter /bin/go-plugin-counter
 
 ENTRYPOINT [ "/bin/go-plugin-counter" ]
 
-FROM docker.mirror.hashicorp.services/ubuntu as nonroot
+FROM docker.mirror.hashicorp.services/ubuntu@sha256:2e863c44b718727c860746568e1d54afd13b2fa71b160f5cd9058fc436217b30 as nonroot
 
 COPY go-plugin-counter /bin/go-plugin-counter
 
 RUN apt-get update && apt-get install -y libcap2-bin acl && \
-    addgroup --system nonroot && \
-    adduser --system --ingroup nonroot nonroot && \
+    groupadd --system nonroot && \
+    useradd --system -g nonroot nonroot && \
     chown -R nonroot:nonroot /bin/go-plugin-counter && \
+    chmod 0755 /bin/go-plugin-counter && \
```

Comment:
I think the error running this was due to gVisor, so I think this chmod can be removed.

```suggestion
```

Reply:
Removed, thanks!